internal/indexer/definitions/torrentday.yaml

```diff
@@ -0,0 +1,58 @@
+---
+#id: torrentday
+name: TorrentDay
+identifier: td

```

Comment:
Would like to have the full names for identifiers, so please change to `torrentday`.

Reply:
done